README.md

```diff
@@ -256,28 +254,48 @@ If you wish to watch the config file and make updates to your configuration, use
 	// additional sources can be passed along with the watching file source and the
 	// precedence order will still be dictated by the order in which the sources are
 	// defined in the Config function.
-	d, err := dials.Config(ctx, config, watchingFileSource)
+	d, err := dials.Config(ctx, defCfg, watchingFileSource)
 	if err != nil {
 		// error handling
 	}
 
-	conf := d.View().(*Config)
-
-	// you can get notified whenever the config changes through the channel
-	// returned by the Events method. Wait on that channel if you need to take
-	// any steps when the config changes
-	go func(ctx context.Context){
-		for{
-			select{
-			case <-ctx.Done():
-				return
-			case c := <-d.Events():
-				// increment metrics and log
-			}
-		}
+	conf, serial := d.ViewVersion()
+
+	// you can get notified whenever the config changes by registering a callback
```

Comment:
nit: extra indentation
```suggestion
	// You can get notified whenever the config changes by registering a callback.
```

Reply:
Fixed the capitalization.

It was actually worse than extra indentation.  Somehow, I ended up with spaces instead of tabs, and a tabstop of 4 for some of the new lines. I've fixed that.